src/.eslintrc.js

```diff
@@ -23,6 +23,22 @@ module.exports = {
         ]
       }
     },
+    {
+      files: ['govuk/**/*.{cjs,js,mjs}'],
+      excludedFiles: ['govuk/**/*.test.{cjs,js,mjs}'],
+      parserOptions: {
+        // Note: Allow ES6 for import/export syntax (although our code is ES3 for legacy browsers)
+        ecmaVersion: '2015'
+      },
+      plugins: ['es-x'],
+      extends: ['plugin:es-x/restrict-to-es3'],
+      rules: {
+        // We use ES6 modules to structure our code
```

Comment:
Want to mention Rollup on this line too?

```suggestion
        // Rollup transpiles modules to AMD export/define
```